cameras:
    camera1:
        frame_id: center_camera
        height: 1200
        width: 1920
        type: pinhole_radtan
        intrinsics: [2109.75, 2071.72, 949.828, 576.237]
        distortion_coeffs: [-0.108144, 0.138668, -0.003797, -0.004841, 0.0]
transforms:
    transform1:
        frame_id: center_camera
        child_frame_id: top_center_lidar
        translation: [-0.0322306, -0.352079, -0.574468]
        rotation: [0.5 , -0.5 , 0.5, 0.5] # x,y,z,w
